src/vehicle.cpp

```diff
@@ -4312,6 +4312,21 @@ void vehicle::slow_leak()
     }
 }
 
+int vehicle::slowdown( const bool should_fall ) const
+{
+    int relative_dir = std::abs( face.dir() - move.dir() ) % 180;
```

Comment:
Despite being correct in the end (due to `sin`), this math is misleading. For example, `(190 - 0) % 180` gives you `10`, when you actually want `170`. You could simply drop `% 180` and replace the whole thing with `std::abs( sin( DEGREES( face.dir() - move.dir() ) ) )`. The variable can also be made `const`.

Reply:
Fixed